frontend/src/styles/components.css

```diff
@@ -1,25 +1,89 @@
+/* sidebar */
```

Comment:
Pretty much all of these CSS comments aren't necessary, you can figure out what they're doing from their class names. The comments just clutter the code really